src/frontend/src/eth/services/balance.services.ts

```diff
@@ -10,11 +10,12 @@ import { toastsError } from '$lib/stores/toasts.store';
 import type { OptionEthAddress } from '$lib/types/address';
 import type { NetworkId } from '$lib/types/network';
 import type { Token, TokenId } from '$lib/types/token';
+import type { SuccessOrNot } from '$lib/types/utils';
 import { replacePlaceholders } from '$lib/utils/i18n.utils';
 import { isNullish } from '@dfinity/utils';
 import { get } from 'svelte/store';
 
-export const reloadBalance = async (token: Token): Promise<{ success: boolean }> => {
+export const reloadBalance = async (token: Token): Promise<SuccessOrNot> => {
```

Comment:
Just an idea if you like it, instead of `SuccessOrNot` what about calling it `SomethingResult` or `ResultSomething`? Like `ResultSuccess`.

Reply:
naming! ahahaha! I like `ResultSuccess`, I will use that